cores/piduino/WString.h

```diff
@@ -123,5 +123,8 @@ class StringSumHelper : public String
   StringSumHelper(double num) : String(num) {}
 };
 
+typedef char __FlashStringHelper;
+#define PROGMEM
+
```

Comment:
Awesome ... do you wish me to fix this up in this pull-request or commit and add a new issue to fix in the next pull-request?